newIDE/app/src/AssetStore/index.js

```diff
@@ -452,44 +442,6 @@ export const AssetStore = React.forwardRef<Props, AssetStoreInterface>(
       [shopNavigationState, saveScrollPosition]
     );
 
-    // If the user has received the pack they are currently viewing,
-    // we update the window to show it if they are not already on the pack page.
-    React.useEffect(
```

Comment:
This is not needed anymore as the Information Page now shows correctly that the pack is "Owned" with a button to browse the assets. (better ux than redirecting to the assets list and having to wait for them to load)

Reply:
Good riddance